modules/smtp/site.js

```diff
@@ -381,3 +390,264 @@ function smtpSettingsPageHandler() {
         );
     });
 }
+$(function () {
+    if (!hm_is_logged()) {
+        return;
+    }
+    if (hm_page_name() === 'settings') {
```

Comment:
Please resolve this thread: https://github.com/cypht-org/cypht/pull/1091/files#r1818718989